c2corg_ui/templates/article/edit.html

```diff
@@ -109,6 +110,46 @@ <h2 class="heading show-phone" translate>general informations</h2>
           </div>
         </section>
 
+        ## ASSOCIATIONS
+        <section class="section associations">
+          <h2 class="heading show-phone"><span translate>associations</span></h2>
+          <h5 class="full" translate ng-show="editCtrl.documentService.document.associations.waypoints.length == 0 &&
+                                              editCtrl.documentService.document.associations.routes.length == 0">
+            You can add associated documents by searching in the field.
+          </h5>
+          <div class="content route-associations">
+            <app-simple-search parent-id="article.document_id"
+              app-select="editCtrl.documentService.pushToAssociations(doc)" dataset="wrcb"></app-simple-search>
+            ${show_editing_associated_waypoints('article')}
+            ${show_editing_associated_routes('article')}
+            ${show_editing_associated_articles('article')}
+            ${show_editing_associated_books('article')}
+          </div>
+
+          <div id="participants-group" class="form-group  full">
+            <label translate>associated_users</label>
```

Comment:
What if the user decides to change his personal article to collaborative - should all the user associations be deleted on API automatically in that moment? Maybe I should not allow it here if we aren't sure about this because it will just result in many user associations?